src/integrationTest/resources/application.properties

```diff
@@ -0,0 +1,30 @@
+scheduling.task.delete-envelopes-dlq-messages.enabled=false
+scheduling.task.consume-envelopes-queue.enabled=false
+
+hystrix.command.default.metrics.rollingStats.timeInMilliseconds=500
+hystrix.command.default.metrics.rollingStats.numBuckets=5
+hystrix.command.check-jurisdiction-log-in.circuitBreaker.sleepWindowInMilliseconds=100000
```

Comment:
do we need this for int tests ?

Reply:
left the differences compared to main app config. didn't pay too much attention to other configs while doing this pr